assets/js/app/Panel.js

```diff
@@ -13,15 +13,15 @@
 
 */
 
-LocusZoom.Panel = function() { 
+LocusZoom.Panel = function(id, layout) { 
 
     this.initialized = false;
     
-    this.id     = null;
+    this.id     = id;
     this.parent = null;
     this.svg    = {};
-    
-    this.view = {
+
+    this.layout = layout || {
```

Comment:
Again, it might be nice to support partial layout specification. So you could say `{height:300}` and have everything else be default. But actually I guess that no one calls this constructor directly? Is it all done via addPanel? So maybe this comment is irrelevant for this code chunk, but then maybe layout should be required?